tests/testthat/test-file.R

```diff
@@ -0,0 +1,135 @@
+
+context("file-based keyring")
+
+test_that("specify keyring explicitly", {
+
+  service_1 <- random_service()
+  username <- random_username()
+  password <- random_password()
+  keyring <- random_keyring()
+
+  kb <- backend_file$new(keyring = keyring)
```

Comment:
Where does this put the YAML  file? We need to put it in `tempdir()` for the test cases.